packages/bpk-component-image/src/BpkBackgroundImage.tsx

```diff
@@ -71,6 +73,18 @@ class BpkBackgroundImage extends Component<BpkBackgroundImageProps> {
     }
   }
 
+  componentDidUpdate(prevProps: BpkBackgroundImageProps) {
```

Comment:
there is `UNSAFE_componentWillReceiveProps` would also trigger `startImageLoad`, maybe can refactored to consolidate the logic into `componentDidUpdate` removing redundancy.